src/SUMMARY.md

```diff
@@ -21,21 +21,24 @@
     * [Write Document](contribute-to-tidb/write-document.md)
 
 * [Understand TiDB](understand-tidb/introduction.md)
-    * [MySQL protocol & session management](understand-tidb/mysql-protocol-and-session-management.md)
-    * [The lifecycle of a SQL](understand-tidb/the-lifecycle-of-a-sql.md)
-    * [Privilege management](understand-tidb/privilege-management.md)
-    * [DDL](understand-tidb/ddl.md)
-    * [DML](understand-tidb/dml.md)
-    * [Transaction processing](understand-tidb/transaction-processing.md)
-    * [Overview of planner](understand-tidb/overview-of-planner.md)
-    * [Logical optimization & rules](understand-tidb/logical-optimization-and-rules.md)
-    * [Physical optimization](understand-tidb/physical-optimization.md)
-    * [Table statistics](understand-tidb/table-statistics.md)
-    * [Plan cache](understand-tidb/plan-cache.md)
-    * [SQL hints and plan management](understand-tidb/sql-plan-management.md)
-    * [Overview of the execution engine](understand-tidb/overview-of-the-execution-engine.md)
-    * [Retrieving data from the storage layer](understand-tidb/communicate-with-the-storage.md)
-    * [MVCC garbage collection](understand-tidb/mvcc-garbage-collection.md)
+    * [The Lifecycle of a Statement](understand-tidb/the-lifecycle-of-a-statement.md)
+        * [DDL](understand-tidb/ddl.md)
+        * [DML](understand-tidb/dml.md)
+        * [DQL](understand-tidb/dql.md)
+    * [Parser](understand-tidb/parser.md)
+    * [Planner](understand-tidb/planner.md)
```

Comment:
You can regard `[Planner](understand-tidb/planner.md)` as an overview. If you have something to discuss overly, you can populate this section in details, otherwise you can leave it as a simple sub toc.

Reply:
You may take these pages as examples:

* https://pingcap.github.io/tidb-dev-guide/get-started/introduction.html
* https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/introduction.html
* https://pingcap.github.io/tidb-dev-guide/understand-tidb/introduction.html

Reply:
You can approve the PR if there is no further concerns.